gulpfile.js

```diff
@@ -31,9 +36,16 @@ theo.registerFormat('sketchpalette', require('./formats/sketchpalette.js'));
 theo.registerFormat('ase.json', require('./formats/ase.json.js'));
 theo.registerFormat('android.xml', require('./formats/android.xml.js'));
 theo.registerFormat('ios.json', require('./formats/ios.json.js'));
-
 theo.registerFormat('d.ts', require('./formats/d.ts'));
 
+theo.registerFormat('light.yml', tokenify(backOfficeVariants, 'light'));
```

Comment:
So technically this creates the back office light variants, but if I register the format name as `back-office.light.yml` then the dist file name is `back-office.back-office.light.json` or whatever filetype. Maybe I can do some a convention over configuration approach and `tokenify` can expect the token file name to match the variant file name. 🤔